flake.nix

```diff
@@ -154,6 +154,13 @@
     processComposeModules.default =
       import ./nix/external-services.nix { inherit inputs; };
     environments.externalServices = import ./nix/external-services-envs.nix;
+
+    om.ci.default = {
```

Comment:
`om.yaml` preferable, because that makes `om ci` run a bit faster.

Reply:
https://github.com/juspay/nix-common/pull/4/commits/c582e8393482274020f6714532609eec23847e66